transports/realtime-websocket-transport/src/index.ts

```diff
@@ -1,8 +1 @@
-// export * from "./realTimeWebSocketTransport";
-// export * from "../../../lib/wavtools/dist/index.d.ts";
-
-import { WavMediaManager } from "../../../lib/media-mgmt/mediaManager";
-import { DailyMediaManager } from "../../../lib/media-mgmt/dailyMediaManager";
-import { WebSocketTransport } from "./webSocketTransport.ts";
-
-export { WavMediaManager, DailyMediaManager, WebSocketTransport };
+export * from "./webSocketTransport";
```

Comment:
the fact that the folder is realtime-websocket-transport but the transport is just websockettransport is still throwing me.